.travis.yml

```diff
@@ -20,27 +21,18 @@ matrix:
   fast_finish: true
   allow_failures:
     # To allow failures, you need to specify the full environment
-    - env: REQUIREMENTS=devel ES_URL=$ES7_DOWNLOAD_URL EXTRAS=all,postgresql,elasticsearch7
+    - env: REQUIREMENTS=devel EXTRAS=all,postgresql,elasticsearch7 ES_VERSION=ES_7_LATEST
 
 cache:
   - pip
 
-services:
-  - postgresql
-  - redis
-
 env:
   global:
-    - ES6_DOWNLOAD_URL="https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-6.8.2.tar.gz"
-    - ES7_DOWNLOAD_URL="https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-7.3.0-linux-x86_64.tar.gz"
-    - ES_HOST=127.0.0.1
-    - SQLALCHEMY_DATABASE_URI="postgresql+psycopg2://postgres@localhost:5432/invenio"
+    - DOCKER_COMPOSE_VERSION=1.26.2
   matrix:
-    # - REQUIREMENTS=lowest ES_URL=$ES7_DOWNLOAD_URL
-    # - REQUIREMENTS=release ES_URL=$ES7_DOWNLOAD_URL
-    - REQUIREMENTS=release ES_URL=$ES6_DOWNLOAD_URL EXTRAS=all,postgresql,elasticsearch6
-    - REQUIREMENTS=release ES_URL=$ES7_DOWNLOAD_URL EXTRAS=all,postgresql,elasticsearch7
-    - REQUIREMENTS=devel ES_URL=$ES7_DOWNLOAD_URL EXTRAS=all,postgresql,elasticsearch7
+    - REQUIREMENTS=release EXTRAS=all,postgresql,elasticsearch6 ES_VERSION=ES_6_LATEST
```

Comment:
Comment: We should add MySQL as well.

Reply:
Since there is not `model` defined here we did not find it necessary. `Invenio-RDM-Records` should do the checking of the different databases?